nilearn/image/__init__.py

```diff
@@ -4,11 +4,10 @@
 """
 from .resampling import resample_img, reorder_img
 from .image import high_variance_confounds, smooth_img, crop_img, \
-            mean_img, swap_img_hemispheres
+    mean_img, swap_img_hemispheres, index_img, iter_img
 from .._utils.niimg_conversions import concat_niimgs as concat_imgs
 
 __all__ = ['resample_img', 'high_variance_confounds', 'smooth_img',
            'crop_img', 'mean_img', 'reorder_img',
-           'swap_img_hemispheres', 'concat_imgs']
-
-
```

Comment:
That might have been a bit harsh on you indeed ;-). In my defense I added a line in the immediate vicinity so I was allowed to fix this PEP8 infraction.